src/tab.js

```diff
@@ -187,3 +187,45 @@ let updateAction = (tabId, delay = null) => {
     });
   }, delay));
 };
+
+
+util.addListener(browser.commands.onCommand, async (command) => {
+  try {
+    switch (command) {
+      case 'toggle-tab-status':
+        await toggleTabStatus();
+        break;
+      case 'toggle-global-status':
+        await toggleGlobalStatus();
+        break;
+      case 'open-options':
+        await openOptionsPage();
+        break;
```

Comment:
This can be simplified without additional functions

```suggestion
      case 'toggle-global-status':
        await util.setSettings({ global_status: !settings.global_status });
        break;
      case 'open-options':
        await browser.runtime.openOptionsPage();
        break;
```